feature/validate-subject-pool/src/main/res/layout/fragment_validate_subject_pool.xml

```diff
@@ -71,7 +71,20 @@
             android:layout_width="match_parent"
             android:layout_height="2dp"
             android:background="@color/simprints_grey_light"
-            app:layout_constraintBottom_toTopOf="@id/validationActionsClose" />
+            app:layout_constraintBottom_toTopOf="@id/validationActionsContinue" />
+
+
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/validationActionsContinue"
+            style="@style/Widget.Simprints.Button.TextButton.White"
+            android:layout_width="0dp"
+            android:layout_height="96dp"
+            android:layout_marginTop="2dp"
+            android:text="@string/id_pool_validation_action_continue"
+
```

Comment:
Nit-pick: couple of random unnecessary empty lines.

Reply:
done